package.json

```diff
@@ -58,6 +59,7 @@
     "flagged": "^2.0.1",
     "focus-visible": "^5.2.0",
     "history": "^5.0.0",
+    "immutable": "~3.7.4",
```

Comment:
Uhm...? Can you remove those?

Reply:
The project doesn't build without these two packages now.

Reply:
See https://github.com/google/web-stories-wp/runs/1839872700#step:10:295

Reply:
Then the `package-lock.json` got messed up or something. There's no reason why these would be suddenly required.

Try resetting the lock file with `git checkout main package-lock.json` and then run `npm install` again.

Reply:
I am having weird issues with running npm and it no longer works to just delete the package-lock.json file, package.json can not reslove the local pacakges. I think we need to look at this issue at some point, in another issue.